stdlib/public/Concurrency/TaskPrivate.h

```diff
@@ -438,6 +438,40 @@ inline bool AsyncTask::localValuePop() {
   return _private().Local.popValue(this);
 }
 
+/*************** Methods for Status records manipulation ******************/
+
+/// Remove a status record from a task.  After this call returns,
+/// the record's memory can be freely modified or deallocated.
+///
+/// This must be called synchronously with the task.  The record must
+/// be registered with the task or else this may crash.
+///
+/// The given record need not be the last record added to
+/// the task, but the operation may be less efficient if not.
+///
+/// Returns false if the task has been cancelled.
+SWIFT_EXPORT_FROM(swift_Concurrency)
+SWIFT_CC(swift) bool swift_task_removeStatusRecord(TaskStatusRecord *record);
```

Comment:
If these are no longer public ABI, (1) they shouldn't be exported and (2) please give them names that don't sound like they're in the ABI.  For example, this one could just be `removeTaskStatusRecord` without any prefixes.

Reply:
Ah. Okay I didn't realize the `swift_` prefix was for exported functions only. I'll adjust them.

Reply:
So while I was fixing this, I realized that we do want to be able to export them from the concurrency library to be able to write unit tests against them - see `unittests/runtime/TaskStatus.cpp`.

So shall I keep to the regular `swift_` prefixed names or should we do something else since we're talking about exporting for the sake of testing?